chef/data_bags/crowbar/migrate/crowbar/102_chef-solr-heap.rb

```diff
@@ -0,0 +1,10 @@
+def upgrade(ta, td, a, d)
+  a["chef"] = ta["chef"]
+  a["chef"]["solr_heap"] = ta["chef"]["solr_heap"]
```

Comment:
(that being said, with a backport to cloud 6, we'll need to check if the attributes already exist -- we do that in some migrations already)